microsoft-edge/progressive-web-apps-chromium/webappmanifests.md

```diff
@@ -10,21 +10,37 @@ ms.technology: pwa
 keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft Store
 ---
 
-# Understand web app manifests in Progressive Web Apps
-
-You can control several settings of your PWA by adding sections to the Web App Manifest...
-
-## Add shortcuts
-
-Use shortcuts to ... The following code shows how to add a shortcut to your web app manifest. 
-
-
-```js
-// code snippet
+# Deeper OS integrations via the Web App Manifest
+
+A site’s Web App Manifest governs how it looks and behaves when installed on a device. At its most basic, the Manifest provides detail about what name and icons should be used to represent your app in system menus to the theme colors the OS should use for the title bar. The Manifest also enables you to unlock more powerful features that allow your app to behave like other native apps on the system.
+
+## Provide quick access to key features
+
+Most operating systems provide some sort of quick access to key app features via a context menu connected to the app’s icon. This feature is, called "shortcuts," is enabled in Progressive Web Apps via the Web App Manifest’s `shortcuts` member. Shortcuts are defined in the manifest like this:
+
+```json
+  "shortcuts": [
+    {
+      "name": "Play Later",
+      "description": "View the list of podcasts you saved for later",
+      "url": "/play-later",
+      "icons": [
+        {
+          "src": "/icons/play-later.svg",
+          "type": "image/svg+xml",
+          "purpose": "any"
+        }
+      ]
+    },
+    {
+      "name": "Subscriptions",
+      "description": "View the list of podcasts you listen to",
+      "url": "/subscriptions?sort=desc"
+    }
+  ]
 ```
 
-(Additional information example)
-The above code can be used to do...if you plan to do, consider using...
+In the context of a full Web App Manifest, adding the above snippet would enable two shortcuts on the app’s icon. The first would be labelled "Play Later" and would have a custom icon. The second would be labelled "Subscriptions" and does not have an icon associated with it. As you can see, the `icons` member is optional. The `description` member is also optional and can be used to provide additional information for accessibility purposes.
```

Comment:
```suggestion
When added to a complete Web App Manifest, adding the above snippet would enable two shortcuts on the context menu on the app’s icon. The first would be labelled "Play Later" and has a custom icon. The second would be labelled "Subscriptions" and does not have an icon because the `icons` member is optional. The `description` member is also optional and may be used to provide additional information for accessibility purposes.
```